Code/6-wire-version/data/mqtt.html

```diff
@@ -2,12 +2,12 @@
 <html>
 <head>
     <title>Lay-Z-Spa Module | MQTT Config</title>
-	<link rel="icon" type="image/png" sizes="300x300" href="favicon.png">
+	<link rel="icon" type="image/png" sizes="48x48" href="favicon.png">
```

Comment:
What does this sizes do?

Reply:
guess for .ico files where you have more than just one size in one file. but in main, the size of the image
https://www.w3schools.com/tags/att_link_sizes.asp

Reply:
Okay